doc/script_commands.txt

```diff
@@ -5340,6 +5340,30 @@ Check getitem2() to understand the arguments of the function.
 
 ---------------------------------------
 
+*countnameditem(<item id>, "<character name>")
+*countnameditem(<item id>, <character ID>)
+*countnameditem("<item name>", "<character name>")
+*countnameditem("<item name>", <character ID>)
+
+This function will return the number of signed items for the specified item ID
+that the invoking character has in their inventory.
+
+	mes("[Item Checker]");
+	if (countnameditem(Apple) > 0) {
+		mes("You have an apple with your name on it!");
+	} else {
+		mes("You do not have an apple with your name on it.");
+	}
+	next;
+	mes("[Item Checker]");
+	mes("You have "+countnameditem(Apple, "John")+" apples with John's name on it!");
```

Comment:
If you want support for translations, you cant split sentenses.
Need use sprintf or other formatting functions.

Example: ``mes(sprintf(_$("You have %d apples with John's name on it!"), countnameditem(Apple, "John"))``

But this is docs, because this i not sure is it really need here.

Reply:
Or 
```suggestion
mesf("You have %d apples with John's name on it!", countnameditem(Apple, "John"));
```

Reply:
I agree that docs should show the recommended code style, since this is where people learn/copy from. The suggested `mesf()` variant is the cleanest/most appropriate